app/Http/Controllers/ReservationController.php

```diff
@@ -46,11 +46,22 @@ public function acceptReject(Request $request)
     {
         $hostNumber = $request->input('From');
         $smsInput = strtolower($request->input('Body'));
-        $host = User::where(DB::raw("CONCAT('+',country_code::text, phone_number::text)"), 'LIKE', "%".$hostNumber."%")
-                    ->get()
-                    ->first();
+
+        $connection = config('database.default');
+        $driver = config("database.connections.{$connection}.driver");
+        if ($driver === 'sqlite') {
+            $concat_string = DB::raw("'+' || country_code || phone_number");
+        } else {
+            $concat_string = DB::raw("CONCAT('+',country_code::text, phone_number::text)");
+        }
```

Comment:
This query is trying to find a concatenation of columns. Didn't find a better way because sqlite doesn't support `CONCAT`. It's moved to the Model for now as suggested.